css/main.css

```diff
@@ -0,0 +1,1623 @@
+}
+
+.f2-a2 {
+  background-color: rgb(246, 243, 243);
+  width: 270px;
+  height: 81px;
+  margin-top: -1px;
+}
+
+.f2-a3 {
+  background-color: rgb(246, 243, 243);
+  width: 270px;
+  height: 87px;
+  font-size: 12px;
+  line-height: 20px;
+  border-top: 1px solid rgb(255, 68, 34);
+}
+
+.f2-icon {
+  width: 16px;
+  height: 16px;
+  padding-left: 15px;
+}
+
+.f2-txtalign {
+  padding-top: 10px;
+  padding-right: 5px;
+}
+
+.flex-1 {
+  display: flex;
+}
+
+.f2-textarea {
+  margin-left: 17px;
+  color: rgb(85, 85, 85);
+}
+
+/* section 3 */
+
+.f3-grid {
+  display: block;
+}
+
+.f3 {
+  width: 100%;
+  height: 679px;
+  border: 1px solid transparent;
+}
+
+.f3-link {
+  position: relative;
+  top: -15px;
+}
+
+.f3-first {
+  width: 100%;
+  height: 276px;
+  line-height: 21px;
+  text-decoration: none;
+  position: relative;
+  top: -25px;
+  font-size: 12px;
+}
+
+.f3-background-1 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/bitcoin_today-header_bitcoin_today-368x276.jpg");
+  background-size: cover;
+}
+
+.f3-background-2 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/04/Social-media-hed-image-368x276.jpg");
+  background-size: cover;
+}
+
+.f3-background-3 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/header-image-battery-life-of-google-pixel-phones-comparison-368x276.png");
+  background-size: cover;
+}
+
+.f3-background-4 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2018/07/weed_tech-368x276.jpg");
+  background-size: cover;
+}
+
+.f3-background-5 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/a-female-office-worker-at-the-table-holding-papers-with-diagrams-368x276.jpg");
+  background-size: cover;
+}
+
+.f3-background-6 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/106909103_10101777330197904_2889321509499508446_o-1-368x276.jpg");
+  background-size: cover;
+}
+
+.f3-background-7 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/corona_news-6_blue-368x276.gif");
+  background-size: cover;
+}
+
+.f3-background-8 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/BMW-ix3-unveiling-car-future-bavarian-motor-ev-electric-368x276.png");
+  background-size: cover;
+}
+
+.f3-background-9 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/acquisition-blog-post-thisstartupannouncementdoesntexist-368x276.png");
+  background-size: cover;
+}
+
+.f3-img-text {
+  text-decoration: none;
+  line-height: 25px;
+  position: relative;
+  top: 150px;
+  padding-left: 6px;
+}
+
+.linear-gradient {
+  background: rgba(250, 0, 0, 0.25) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+  background-size: 100% 100%;
+  height: 200px;
```

Comment:
- To fix vertical alignment in **mobile** in your big images, please use `height: 270px;`